.github/workflows/unittests.yml

```diff
@@ -59,6 +59,10 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci-skip')"
     steps:
+      - name: Cancel Outdated Runs
```

Comment:
I added this as a normal step to one of the CI jobs, however we could also run this on a dedicated job (however then it would showup in the PR checks summary).